src/rabbit_delayed_message.erl

```diff
@@ -260,3 +267,17 @@ recover_exchange_and_bindings(#exchange{name = XName} = X) ->
                               "recovered bindings for ~s",
                              [rabbit_misc:rs(XName)])
     end).
+
+bump_routed_stats(ExName, Qs) ->
+    rabbit_global_counters:messages_routed(amqp091, length(Qs)),
```

Comment:
is that correct that the Protocol is set to `amqp091`? at this point we don't know what the original protocol was.

Reply:
We don't but custom exchanges are rarely used via STOMP, so we can make this assumption.